readme.md

```diff
@@ -1,87 +1,130 @@
 # ovsrpro
 
-Build Overseer [dependencies](projects/README.md) by leveraging externpro
-(https://github.com/smanders/externpro).
+Build Overseer [dependencies](projects/README.md) by leveraging
+[externpro](https://github.com/smanders/externpro).
 
 ## Installing ovsrpro
 
 1. Download the installer for the release you need, for the your development
    environment (OS, compiler, etc). See
-   https://github.com/distributePro/ovsrpro/releases.
+   <https://github.com/distributePro/ovsrpro/releases>.
 1. Download the SHA256 checksum file for your release.
 1. Validate the checksum.
+   ```bash
+   sha256sum --check ovsrpro-18.10.1-gcc631-64-Linux.sh.sha256
+   ```
 1. Run the installer.
+   ```bash
+   sudo ./ovsrpro-18.10.1-gcc631-64-Linux.sh --prefix=/opt/extern/ --include-subdir
+   ```
+   The recommended location for installation is */opt/ovsrpro/*. We also
+   recommend using the subdirectory. Both these options are in this example.
+
+## Using ovsrpro
+
+Copy the file *share/cmake/Findovsrpro.cmake* from your installation to your
+project's root directory. In your *CMakeLists.txt* add these lines:
+
+```cmake
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}")
+find_package(ovsrpro 18.10.1 EXACT REQUIRED)
+```
 
+A few notes about the `find_package()` command:
+
+1. The version is required in the command.
+1. `EXACT` is required. If it's omitted, it is forced by *Findovsrpro.cmake*
+   and a warning is issued.
+
+If ovsrpro is found, the following variables are set:
+
+<!-- markdownlint-disable MD013 -->
+| Variable | Description |
+|:---|:---|
+| `ovsrpro_FOUND` | This is set to `true` if ovsrpro is found, and `false` if it not. |
+| `ovsrpro_ROOT_DIR` | This is set to the root path where ovsrpro is installed. An example is */opt/ovsrpro/ovsrpro-20.04.1-gcc921-64-Linux/* |
+| `ovsrpro_INCLUDE_DIR` | This is the root path to the project headers files. Each project will have a separate subdirectory within this path. An example is */opt/ovsrpro/ovsrpro-20.04.1-gcc921-64-Linux/include/*. Ovsrpro users typically should not need to use this. |
+| `ovsrpro_LIBRARY_DIR` | This is the path to the project library files. An example is */opt/ovsrpro/ovsrpro-20.04.1-gcc921-64-Linux/lib/*. Ovsrpro users typically should not need to use this. |
+| `ovsrpro_CONFIG_PATH` | This the path to find the Qt5 CMake find scripts. This is needed to find Qt5 using `find_package()`. |
+<!-- markdownlint-disable MD013 -->
+
+`${ovsrpro_ROOT_DIR}`*/share/cmake* is appended to `CMAKE_MODULE_PATH`
+automatically. To find ovsrpro projects, except for Qt5, add `find_package()`
+commands. The project names must all be prefixed with "Ovsrpro".
```

Comment:
Is this last line still true? I didn't see where "Ovsrpro" was required when finding the different projects, and it also isn't used in the examples below.

Reply:
Nope. Thanks for pointing this out.